### ISSUE TITLE
Zooniverse(url) links in talk reuse current window/tab on middle-mouse-click/+CTRL key.

### ISSUE THREAD

**Issue by MvGulik** [zooniverse/Panoptes-Front-End#3048]
## Expected behavior

Open clicked link, with middle-mouse-button, in a new tab.
(_Same for clicked link (left-mouse-button), while CTRL key is down._)
## Current behavior

Opens clicked link, with middle-mouse-button, in current tab/window.
(_Same for clicked link (left-mouse-button), while CTRL key is down._)
## Steps to replicate

Middle mouse click on a "`https://www.zooniverse.org/...`" link in a Talk post.
(_checked at 'Gravity Spy' and main zooniverse Talk._) 
## Additional information

Only related to links pointing to a zooniverse url.
Seems, so far, only related to links in Talk.
Seems a fairly resent behavior change.
_Links pointing outside the zooniverse domain do open in a new tab/window._
_Currently the only option to force a zooniverse-url-link to open in a new tab seems to be the trough the "open in new tab" mouse-menu option._
- **Operating system:**
- **Browser:**

**Comment by eatyourgreens** [comment]
Side effect of #3021

**Comment by MvGulik** [comment]
Closing.
Not reproducible anymore.
Assuming/probably fixed by: #3282

**Comment by MvGulik** [comment]
Correction + Reopening.

Seems the for-mentioned patch (3282) fixed it for links in Talk that ... pointed to other Talk posts.
But after hitting on it again, it became clear some others link-types are still effected by this.
```
++ https://www.zooniverse.org/talk/13/18624?comment=164688
++ https://www.zooniverse.org/talk/13/18624
-- https://www.zooniverse.org/talk/13
-- https://www.zooniverse.org/talk
-- https://www.zooniverse.org/about
-- https://www.zooniverse.org
```
